applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_harmonic_analysis_solver.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
looks fine to me. However @loumalouomega can u take a look when you have some time?